.github/workflows/go-build.yaml

```diff
@@ -15,23 +15,22 @@
 name: Build
 
 on:
+  push:
```

Comment:
Can you drop the -pr from all files? I added it thinking we had to use the pull event but no longer.

Reply:
I already did :)

Reply:
Oh you did. Noce